main_net_config.conf

```diff
@@ -127,19 +159,18 @@ node {
   # validateSignThreadNum = 16
 
   maxConnections = 30
-  
+
   minConnections = 8
-  
+
   minActiveConnections = 3
-  
+
   maxConnectionsWithSameIp = 2
 
   maxHttpConnectNumber = 50
 
   minParticipationRate = 15
 
-  receiveTcpMinDataLength = 2048
-  isOpenFullTcpDisconnect = true
+  isOpenFullTcpDisconnect = false
```

Comment:
why  is false ?

Reply:
This function is to randomly disconnect the TCP connection, the current verification function is not very good, so it is set to `false`, and we will consider removing this function in the future.